Formula/zssh.rb

```diff
@@ -29,11 +30,17 @@ class Zssh < Formula
   end
 
   def install
+    system "autoreconf", "--verbose", "--install", "--force"
     system "./configure", "--disable-debug", "--disable-dependency-tracking",
                           "--prefix=#{prefix}"
     system "make"
 
     bin.install "zssh", "ztelnet"
     man1.install "zssh.1", "ztelnet.1"
   end
+
+  test do
+    system "#{bin}/zssh", "--version"
+    system "#{bin}/ztelnet", "--version"
+  end
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
I am aware that just tests running `--version` are not great, although they are very common in Homebrew.  At least they are verify that the compile generated a runnable binary.

As I've said elsewhere, I am just trying to help out fixing bottling issues on Big Sur.  I've never used this particular package and have no idea how to test it more in-depth; I am just trying to make it build again.

Anyway I'll remove this test and we'll just leave this formula test-less as it was before.  It was just an attempt at a slight improvement while I was fixing something else.